repo/fsrepo/fsrepo.go

```diff
@@ -379,6 +398,61 @@ func (r *FSRepo) openDatastore() error {
 	return nil
 }
 
+var SpecFn = "spec"
```

Comment:
sure, `datastore_id` works for me. (i think 'id' is the appropriate term as we are using it as a unique identifier for the on-disk datastore configuration)